src/Input.js

```diff
@@ -8,6 +8,9 @@ class Input extends InputBase {
     if (this.props.type === 'static') { // eslint-disable-line react/prop-types
       deprecationWarning('Input type=static', 'StaticText');
       return <FormControls.Static {...this.props} />;
+    } else if (this.props.type === 'radio') {
+      deprecationWarning('Input type=radio', 'FormControls.RadioButton and FormControls.RadioGroup');
+      return <FormControls.RadioButton {...this.props} />;
```

Comment:
1. Docs will be updated.
2. That my be so, but using `FormGroup` on `RadioButton` is causing all sorts of headache, mainly with inline styles.